explorer/graphql/app/schema.graphql

```diff
@@ -127,4 +127,5 @@ type Query {
   xmsgrange(from: BigInt!, to: BigInt!): [XMsg]!
   xmsgcount: BigInt
   xreceiptcount: BigInt
+  xmsg(sourceChainID: BigInt!, destChainID: BigInt! streamOffset: BigInt!): XMsg
```

Comment:
If `streamOffset` is missing, should we return latest message?

Reply:
Similarly, IMO,  the `xblock` query above should accept a nullable `height` in which case it should return the latest block - this is what Ethereum does.

Reply:
I'll update this